packages/edit-post/src/components/header/plugin-more-menu-item/test/__snapshots__/index.js.snap

```diff
@@ -23,16 +23,16 @@ exports[`PluginMoreMenuItem renders menu item as button properly 1`] = `
     >
       <svg
         aria-hidden="true"
-        className="dashicon dashicons-smiley"
+        className="components-menu-items__item-icon"
         focusable="false"
         height={20}
         role="img"
-        viewBox="0 0 20 20"
```

Comment:
Do we have changes that adjust CSS for a different size?

Ignore if yes. I’m on mobile and it’s hard to scroll 😅

Reply:
I don't get the question :) I changed the icon here so it's a different viexbox

Reply:
Lol, it's hard to explain. Dashicons have a different size than SVG icon but I think it's handled by the fact that you no longer use Dashicon here. Ignore me :)

Reply:
Yes, for dashicons that are still used we use a trick in viewport (-2) to center them properly even if they're smaller.